### DISCUSSION TITLE
Can the object-cache.php class be used without the entire plugin?

### DISCUSSION THREAD

**Discussion by hj-collab** [rhubarbgroup/.github#12] [Support (ROC)] [answered]
Hi @tillkruss,

I just want to use the class file and not the entire plugin. Will there be any issues? Please let me know. Thank you!

Best regards,

**Comment by tillkruss** [comment, marked as answer]
You'll loose visibility into whether the cache is working or not, you won't have analytics and depending how you set it up, you won't get updates to the drop-in.
